### PR TITLE
Partial Credit Update

### PR DESCRIPTION
**Pull request by mhoeber**
UI and provisional note for partners

@lamagnifica

### CODE REVIEW

File: en_us/shared/exercises_tools/Section_partial_credit.rst [hunk 1]
```diff
@@ -17,6 +17,14 @@ By awarding partial credit for problems, you can motivate learners who have
 mastered some of the course content and provide a score that accurately
 demonstrates their progress.
 
+.. only:: Partners
+
+  .. note:: 
+    Support for partial credit problems in courses on edx.org and edX
+    Edge is provisional. Ensure you test such problems thoroughly before
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/Section_partial_credit.rst]: Ensure ^that you

File: en_us/shared/exercises_tools/checkbox.rst [hunk 2]
```diff
@@ -384,6 +384,14 @@ who submit an answer that is partly correct. You must use the `Advanced Editor
 <Use the Advanced Editor to Edit a Checkbox Problem>`_ to configure partial
 credit.
 
+.. only:: Partners
+
+  .. note:: 
+    Support for partial credit problems in courses on edx.org and edX
+    Edge is provisional. Ensure you test such problems thoroughly before
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/checkbox.rst]: ^that

File: en_us/shared/exercises_tools/custom_python.rst [hunk 3]
```diff
@@ -352,6 +352,14 @@ give a partially correct answer receive 50% of the points for a problem. To
 provide a learner with a more granular score, see `Award a Percentage of
 Credit for the Problem`_.
 
+.. only:: Partners
+
+  .. note:: 
+    Support for half credit Python-evaluated input problems in courses on
+    edx.org and edX Edge is provisional. Ensure you test such problems
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/custom_python.rst]: yeah

File: en_us/shared/exercises_tools/dropdown.rst [hunk 4]
```diff
@@ -340,6 +340,14 @@ You can configure a dropdown problem so that specific incorrect answers award
 learners partial credit for the problem. You must use the `Advanced Editor <Use
 the Advanced Editor to Edit a Dropdown Problem>`_ to configure partial credit.
 
+.. only:: Partners
+
+  .. note:: 
+    Support for partial credit problems in courses on edx.org and edX
+    Edge is provisional. Ensure you test such problems thoroughly before
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/dropdown.rst]: yep

**Comment by lamagnifica** [comment]
Does partial-credit-python-problem.png need to be updated too? or does that still get the check mark?
otherwise, looks good to me :+1:

**Comment by mhoeber** [comment]
@lamagnifica thanks.  the python image is ok - was for the old/different partial credit problem

**Comment by catong** [comment]
Reverting this PR and https://github.com/edx/edx-documentation/pull/513, per @explorerleslie, due to issue in Sept 2 release. CC @mhoeber

**Comment by explorerleslie** [comment]
:+1:

**Comment by lamagnifica** [comment]
:+1:

**Pull request merged**